client/network/test/src/fuzz.rs

```diff
@@ -294,22 +311,22 @@ fn test_once() {
 						current_peer = Some(peer_id);
 						current_event = Some(Event::PsmReject);
 					},
-					Poll::Ready(None) => panic!(),
-					Poll::Pending => {},
+					Some(None) => panic!(),
+					None => {},
```

Comment:
What is the difference between these?

Reply:
`Some(None)` means that `ProtocolController` has terminated, while `None` means that there is no message currently available to receive from the channel.